notebooks/en/constrained_generation.ipynb

```diff
@@ -0,0 +1,522 @@
+{
```

Comment:
<p>Maybe change to the below keep the language consistent?</p><p> </p><p>"This notebook demonstrates <em>constrained</em> generation with two inference methods:"</p><p> </p><p>I think it'd also be helpful to add a brief description of what "outlines" is, for example:</p><p> </p><p>" Or locally using outlines, a structured text generation library"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/64/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Thank you @stevhliu! </p><p>You comment made me reconsider the structure of the notebook: and actually I will rename it Structured Generation</p><p>Indeed, this broader method is named "structured generation", and can be achieved either via prompting or Constrained generation: given that the notebook demonstrates both methods, it would be more appropriate to name it under the most general Structured Generation.</p>